corehq/apps/domain/views.py

```diff
@@ -538,6 +539,28 @@ def post(self, request, *args, **kwargs):
 
 @require_POST
 @require_can_edit_web_users
+def cancel_repeat_record(request, domain):
+    record = RepeatRecord.get(request.POST.get('record_id'))
```

Comment:
what happens if this RepeatRecord isn't found? I think you should throw a `404`.